Sources/SwiftRefactor/ExpandEditorPlaceholder.swift

```diff
@@ -78,12 +80,30 @@ import SwiftSyntaxBuilder
 /// ```
 struct ExpandSingleEditorPlaceholder: EditRefactoringProvider {
   struct Context {
-    let indentationWidth: Trivia?
-    let initialIndentation: Trivia
-
-    init(indentationWidth: Trivia? = nil, initialIndentation: Trivia = []) {
-      self.indentationWidth = indentationWidth
-      self.initialIndentation = initialIndentation
+    /// The formatter to use when expanding a function-typed placeholder.
+    let closureLiteralFormat: BasicFormat
+    /// When true, the expansion will wrap a function-typed placeholder's entire
+    /// expansion in placeholder delimiters, in addition to any placeholders
+    /// inside the expanded closure literal.
+    ///
+    /// ## With nested placeholders allowed
+    /// ### Before
+    /// ```swift
+    /// <#T##(Int) -> String##(Int) -> String##(_ someInt: Int) -> String#>
+    /// ```
+    ///
+    /// ### After
+    /// ```swift
+    /// <#{ someInt in <#String#> }#>
+    /// ```
```

Comment:
I’m not sure `Before` and `After` are the correct headings here. I would suggest

```suggestion
    /// With `allowNestedPlaceholders = false`
    /// ```swift
    /// <#T##(Int) -> String##(Int) -> String##(_ someInt: Int) -> String#>
    /// ```
    ///
    /// With `allowNestedPlaceholders = true`
    /// ```swift
    /// <#{ someInt in <#String#> }#>
    /// ```
```

Reply:
Sure, was following the wording from the doc comment above this; changed!